<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LR_drawer</class>
 <widget class="QMainWindow" name="LR_drawer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LR_drawer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>220</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Input options:</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>231</width>
           <height>161</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Enter x values:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="x_input">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maxLength">
             <number>3276700</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Enter y values:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="y_input">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maxLength">
             <number>3276700</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="load">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Plot options:</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>221</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Choose a model:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="user_mode_choice">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Linear fit model</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bilinear fit model</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Enter x0:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="x0_choice"/>
          </item>
          <item>
           <widget class="QPushButton" name="Plot">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Plot</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInput">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_file"/>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="title">
     <string>Edit plot</string>
    </property>
    <addaction name="actionChange_tickness"/>
    <addaction name="actionChange_color"/>
   </widget>
   <addaction name="menuInput"/>
   <addaction name="menuStyle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_file"/>
   <addaction name="actionChange_tickness"/>
   <addaction name="actionChange_color"/>
  </widget>
  <action name="actionLoad_file">
   <property name="icon">
    <iconset>
     <normaloff>../img_504113.png</normaloff>../img_504113.png</iconset>
   </property>
   <property name="text">
    <string>Load input</string>
   </property>
  </action>
  <action name="actionChange_tickness">
   <property name="icon">
    <iconset>
     <normaloff>../181398-200.png</normaloff>../181398-200.png</iconset>
   </property>
   <property name="text">
    <string>Change tickness</string>
   </property>
  </action>
  <action name="actionChange_color">
   <property name="icon">
    <iconset>
     <normaloff>../6684101501555931400-512.png</normaloff>../6684101501555931400-512.png</iconset>
   </property>
   <property name="text">
    <string>Change color</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
